xapian-core/api/queryinternal.cc

```diff
@@ -800,7 +803,8 @@ Query::Internal::unserialise(const char ** p, const char * end,
 	    //     0: wqf = 0; pos = 0
 	    //     1: wqf = 1; pos = 0
 	    //     2: wqf = 1; pos -> encoded value follows
-	    //     3: wqf -> encoded value follows; pos -> encoded value follows
+	    //     3: wqf -> encoded value follows;
+	    //	   pos -> encoded value follows
```

Comment:
Wrapping this line isn't necessary and makes the comment a bit less readable, but really the key thing is that it isn't part of the change you're trying to make (i.e. "Add OP_VALUE_GT and OP_VALUE_LT").  So even if this were a desirable change, this PR isn't the place to be making it.